src/Theme_Mod_Command.php

```diff
@@ -121,6 +121,52 @@ public function get( $args = array(), $assoc_args = array() ) {
 
 	}
 
+	/**
+	 * Gets all theme mods.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<mod>...]
+	 * : One or more mods to get.
+	 *
+	 * [--field=<field>]
+	 * : Returns the value of a single field.
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - json
+	 *   - csv
+	 *   - yaml
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Get all theme mods.
+	 *     $ wp theme mod list
+	 *     +------------------+---------+
+	 *     | key              | value   |
+	 *     +------------------+---------+
+	 *     | background_color | dd3333  |
+	 *     | link_color       | #dd9933 |
+	 *     | main_text_color  | #8224e3 |
+	 *     +------------------+---------+
+	 *
+	 */
+	public function list( $args = array(), $assoc_args = array() ) {
```

Comment:
`list` is a protected keyword in more recent versions of PHP, so you cannot use it as a method name.

Use `list_` instead, and then add the `@subcommand list` annotation to the docblock, similar to how it is done here: https://github.com/wp-cli/extension-command/blob/master/src/Plugin_Command.php#L1006-L1008